util/copy.go

```diff
@@ -1,25 +1,20 @@
 package util
 
 import (
-	"bytes"
-	"encoding/gob"
+	"encoding/json"
 
 	"github.com/pkg/errors"
 )
 
 // DeepCopy makes a deep copy of the src value into the copy params
```

Comment:
Can we also call out that it only copies exported fields? Unexported (i.e. lowercase) fields won't be copied.

Reply:
Good catch I wasn't aware this was a limitation.